composer.json

```diff
@@ -19,19 +19,20 @@
         }
     ],
     "require": {
-        "php": "^8.1",
-        "knplabs/knp-snappy-bundle": "^1.7",
-        "myclabs/php-enum": "^1.7",
-        "php-http/message-factory": "^1.1",
-        "sylius/resource-bundle": "^1.9",
-        "sylius/sylius": "~1.13.0 || ~1.14.0",
-        "symfony/messenger": "^5.4.21 || ^6.4"
+        "php": "^8.2",
+        "knplabs/knp-snappy-bundle": "^1.10",
+        "myclabs/php-enum": "^1.8",
```

Comment:
I would say it's high time to drop pseudo enum support and use sylius/sylius style consts (or php enums)

Reply:
We would like to do that. There is another task for that. This will be done in another PR